go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/DATA-DOG/godog v0.7.13
 	github.com/YourFin/binappend v0.0.0-20181105185800-0add4bf0b9ad
+	github.com/blang/semver v0.0.0-20190414102917-ba2c2ddd8906
```

Comment:
@anjannath with https://github.com/code-ready/crc/commit/d5bf2551426cd05bf52faafe20a8a4aeb026672e one I already started using `github.com/Masterminds/semver` since it is much more updated and not maintained by a single person.

Reply:
@praveenkumar Done, using `github.com/Masterminds/semver` now.